### DISCUSSION TITLE
allowed-list of referrer header in nextjs client side

### DISCUSSION THREAD

**Discussion by jofese** [vercel/next.js#35252] [Help]
How are you doing?

I have a question regarding the header referrer. In a pentest analysis to my app in nextjs I was told that it is allowing to load client side resources (images, css, js, etc) with a URL from the header referrer different to the allowed urls. For which I must implement a whitelist of urls that does not allow access to resources if the referrer header is different from the list of allowed urls.

I leave an image of the observation, needing to block resources with a header referrer not allowed. 
![avdi7](https://user-images.githubusercontent.com/7877727/157870524-15e9a7f6-7dae-4b6a-9a19-007f331cbb86.png)

How could I implement this validation from NextJs Client Side?

**Comment by Ihsanasfari** [comment]
I encountered the same issue. Has this issue been resolved?
